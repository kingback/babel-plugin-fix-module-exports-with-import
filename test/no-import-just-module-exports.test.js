const babel = require('babel-core');
const plugin = require('../index');

var example = `
const A = require('a');
// ...
module.exports = A;
`;

it('no-import-just-module-exports', () => {
  const {code} = babel.transform(example, {plugins: [plugin]});
  expect(code).toMatchSnapshot();
});