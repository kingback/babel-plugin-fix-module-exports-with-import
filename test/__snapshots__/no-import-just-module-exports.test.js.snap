// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no-import-just-module-exports 1`] = `
"
const A = require('a');
// ...
module.exports = A;"
`;
